IPython/core/displayhook.py

```diff
@@ -16,6 +16,8 @@
 from traitlets import Instance, Float
 from warnings import warn
 
+from IPython.display import display
```

Comment:
Not sure why this import is here. Could you document that if it needs to stay here?